src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeMarshal.cs

```diff
@@ -63,6 +63,34 @@ static partial class WindowsRuntimeMarshal
 
             dataPtr = IntPtr.Zero;
             return false;
+        }
+
+        /// <summary>
+        /// Tries to get an array segment from the underlying buffer. The return value indicates the success of the operation.
+        /// </summary>
+        /// <param name="buffer">The buffer to get the array segment for.</param>
+        /// <param name="dataPtr">When this method returns, contains the array segment retrieved from the underlying  buffer. If the method fails, the method returns a default array segment.</param>
+        public static bool TryGetArray(
+#if NET
+            [NotNullWhen(true)]
+# endif
+            IBuffer source,
+#if NET
+            [NotNullWhen(true)]

```

Comment:
This is useless, `ArraySegment<T>` is a value type. Remove it.

Reply:
Oh right nice :). Let me remove. I just assumed it was a reference type.